lab_4_retrieval_w_clustering/main.py

```diff
@@ -271,11 +381,20 @@ def __init__(self, db: DocumentVectorDB, n_clusters: int) -> None:
             db (DocumentVectorDB): An instance of DocumentVectorDB class.
             n_clusters (int): Number of clusters.
         """
+        self.__clusters = []
+        self._db = db
+        self._n_clusters = n_clusters
 
     def train(self) -> None:
         """
         Train k-means algorithm.
         """
+        new_centroids = self._db.get_vectors()[:self._n_clusters]
+        self.__clusters = [ClusterDTO(centroid[1]) for centroid in new_centroids]
+        self.run_single_train_iteration()
+        while self._is_convergence_reached(self.__clusters) is False:
```

Comment:
don't use `is False`

Reply:
done (used not True instead, not sure it is what you meant...)